lib/python/picongpu/plugins/plot_mpl/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .energy_histogram_visualizer import Visualizer as \
+    EnergyHistogramVisualizer
+from .phase_space_visualizer import Visualizer as PhaseSpaceVisualizer
+from .png_visualizer import Visualizer as PNGVisualizer
+
+
+__all__ = ["EnergyHistogramVisualizer",
```

Comment:
I would not suffix them with `Visualizer`

Reply:
No, not really. I would also have to read up on that. 
Naming collisions were the main reason for suffixing them this way.

Reply:
Ok, but then let us at least Suffix them with `PlotMPL` or just `MPL`

Reply:
Ok, I changed the suffix to 'MPL'